packages/react-aria-components/stories/Calendar.stories.tsx

```diff
@@ -10,13 +10,32 @@
  * governing permissions and limitations under the License.
  */
 
-import {Button, Calendar, CalendarCell, CalendarGrid, Heading, RangeCalendar} from 'react-aria-components';
-import React from 'react';
+import {Button, Calendar, CalendarCell, CalendarGrid, CalendarStateContext, Heading, RangeCalendar} from 'react-aria-components';
+import React, {useContext} from 'react';
 
 export default {
   title: 'React Aria Components'
 };
 
+function Footer() {
+  const state = useContext(CalendarStateContext);
+  const {setValue} = state;
+  
+  return (
+    <div>
+      <Button 
+        slot={null} 
+        className="reset-button"
+        onPress={() => {
+          // reset value 
+          setValue(null);
+        }}>
+        Reset value
+      </Button>
+    </div>
+  );
+}
+
```

Comment:
Ah looks like typescript is complaining about the potential that state is null here if the CalendarStateContext Provider doesn't exist above it. You can prob fix it like 
```suggestion
function Footer() {
  const state = useContext(CalendarStateContext);
  const setValue = state?.setValue;

  return (
    <div>
      <Button
        slot={null}
        className="reset-button"
        onPress={() => {
          // reset value
          setValue?.(null);
        }}>
        Reset value
      </Button>
    </div>
  );
}
```

or doing something like 
```
  const state = useContext(CalendarStateContext) || {} as CalendarState;
  const {setValue} = state;
```

Reply:
Hello!

Thank you very much for the solution provided. I did as you said. I fixed everything.

But now the problem is that when checking `ci/circleci:test-canary`, the following error occurs
[https://app.circleci.com/pipelines/github/adobe/react-spectrum/19333/workflows/f8dae52f-81af-4112-bf3b-e22f2f376a76/jobs/281883](https://app.circleci.com/pipelines/github/adobe/react-spectrum/19333/workflows/f8dae52f-81af-4112-bf3b-e22f2f376a76/jobs/281883/parallel-runs/2/steps/2-103)

But the fact is that this error does not apply to the test file that I wrote. For my part, I have corrected everything.
Please take a look when you have time.

PR is ready for merge

Reply:
Might be https://github.com/adobe/react-spectrum/pull/6059, I've pulled in main to see if it gets fixed

Reply:
Thank you! So everything is fine and you can already make merges in the main? :)

Reply:
yep, I'll review the changes today and see if I can get another pair of eyes on it as well